### PR TITLE
Custom recovery/alert payload

### PR DESCRIPTION
**Pull request by kimsauce**
## Purpose of this pull request

Azure Function, OpsGenie, MsTeams, Jira docs

## Select the type of change:

- [ ] Minor Changes - Typos, formatting, slight revisions
- [ ] Update Content - Revisions and updating sections
- [x] New Content - New features, sections, pages, tutorials
- [ ] Site and Tools - Updates, maintenance, and new packages for the site, Gatsby, React, etc

### CODE REVIEW

**Review by stacykor — APPROVED** [review]
LGTM

File: docs/alerts/webhook-connections/opsgenie-legacy.md [hunk 1]
```diff
@@ -80,8 +80,9 @@ To create a webhook:
       "SourceURL": "{{SourceURL}}"
     }
     ```
-
-1. Click **Save**.
+1. In the **Recovery Payload** section, you can customize your recovery notification.
```
Comment (angadrandhawa1) [inline docs/alerts/webhook-connections/opsgenie-legacy.md:83]: "1. In the **Recovery Payload** section, you can customize your recovery notification." Not applicable to ServiceNow Legacy. Remove it.

File: docs/alerts/webhook-connections/opsgenie-legacy.md [hunk 2]
```diff
@@ -80,8 +80,9 @@ To create a webhook:
...
+1. To test the connection, click **Test Alert or Test Recovery**. If successful, you'll see a `200 OK` response message.
```
Comment (angadrandhawa1) [inline docs/alerts/webhook-connections/opsgenie-legacy.md:84]: "Test Recovery" not applicable. Remove it.

File: docs/alerts/webhook-connections/datadog.md [hunk 3]
```diff
@@ -28,4 +28,7 @@ You'll need the **Manage connections** [role capability](/docs/manage/users-ro
 1. (Optional) Enter a **Description** for the Connection.
 1. Enter the **URL** for the endpoint.
 1. (Optional) **Custom Headers**, enter up to five comma separated key-value pairs.
-1. Under **Payload**, enter a JSON object in the format required by Datadog. For details on variables that can be used as parameters within your JSON object, see [Webhook Payload Variables](set-up-webhook-connections.md). 
+1. Under **Alert Payload**, enter a JSON object in the format required by Datadog. For details on variables that can be used as parameters within your JSON object, see [Webhook Payload Variables](set-up-webhook-connections.md). 
+1. In the **Recovery Payload** section, you can customize your recovery notification.
```
[the remark this answers is not in the document]
Comment (kimsauce) [reply docs/alerts/webhook-connections/datadog.md:32]: thanks!

**Pull request merged**